### PR TITLE
Align y-axis title to the actual axis

### LINKED ISSUE #194: Y-axis label not aligned properly
#### I am submitting a...

- [ ] Feature request
- [ ] Design defect
- [x] Source code defect
- [ ] Demo/documentation defect
- [ ] Other

#### charts version:
0.9.4

### Issue description
Sandbox: https://codesandbox.io/s/p9pk464p80
Y-axis label has to be center aligned vertically w.r.t Y-axis (not w.r.t whole graph area)

### Screenshot or recording
image

### PR DESCRIPTION
**Pull request by licarijd**
### Updates
- y axis title aligned on the y axis

Closes #194

### CODE REVIEW

File: packages/core/src/base-axis-chart.ts [hunk 1]
```diff
@@ -390,22 +393,25 @@ export class BaseAxisChart extends BaseChart {
...
+		const yAxisLabelWidth = this.innerWrap.select(".axis-label.y").node().getBBox().width;
+
+		const yAxisTitleTranslate = {
+			x: - yAxisCenter + yAxisLabelWidth / 2,
+			y: - (tickHeight + Configuration.scales.tick.heightAddition) * 1.5
+		};
+
+		// Align y axis title on the y axis
+		this.innerWrap.select(".axis-label.y").attr("transform", `rotate(-90) translate(${yAxisTitleTranslate.x}, ${yAxisTitleTranslate.y})`);
+
```
Comment (theiliad) [inline packages/core/src/base-axis-chart.ts]: Shouldn't the above go back into the if statement

File: packages/core/src/base-axis-chart.ts [hunk 2]
```diff
@@ -390,22 +393,25 @@ export class BaseAxisChart extends BaseChart {
...
-			y: - (tickHeight + Configuration.scales.tick.heightAddition) * 1.5
-		};
-
 		// Add y-axis title
 		if (this.innerWrap.select(".axis-label.y").nodes().length === 0 && this.options.scales.y.title) {
 			yAxisRef.append("text")
 				.attr("class", "y axis-label")
-				.attr("transform", `rotate(-90) translate(${yAxisTitleTranslate.x}, ${yAxisTitleTranslate.y})`)
-				.attr("text-align", "center")
+				.attr("alignment-baseline", "middle")
```
Comment (theiliad) [inline packages/core/src/base-axis-chart.ts]: This doesn't seem to do anything, does it?

Comment (licarijd) [reply packages/core/src/base-axis-chart.ts]: Just removed the center text align

File: packages/core/src/base-axis-chart.ts (hunk 1 above)
Comment (licarijd) [reply packages/core/src/base-axis-chart.ts]: Yes

File: packages/core/src/base-axis-chart.ts [hunk 3]
```diff
@@ -390,20 +393,22 @@ export class BaseAxisChart extends BaseChart {
...
+			const yAxisCenter = yAxisRef.node().getBBox().height / 2;
+			const yAxisLabelWidth = this.innerWrap.select(".axis-label.y").node().getBBox().width;
+
+			const yAxisTitleTranslate = {
+				x: - yAxisCenter + yAxisLabelWidth / 2,
+				y: - (tickHeight + Configuration.scales.tick.heightAddition) * 1.5
+			};
+
+		// Align y axis title on the y axis
+		this.innerWrap.select(".axis-label.y").attr("transform", `rotate(-90) translate(${yAxisTitleTranslate.x}, ${yAxisTitleTranslate.y})`);
```
Comment (theiliad) [inline packages/core/src/base-axis-chart.ts]: Misindentation in the above

**Comment by theiliad** [comment]
@zvonimirfras @cal-smith

**Comment by theiliad** [comment]
@licarijd There's a merge conflict now after doing some work on the repo. Are we able to fix that?

**Pull request merged**